.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - jrb/ci
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
```

Comment:
This action is maintained by AWS and essentially just [defines environment variables](https://github.com/aws-actions/configure-aws-credentials/blob/be944dcd2cb7647a4e28e1d4a3116ceb5bb35c41/index.js#L97-L121). We could define these ourselves, but the action has support for session tokens and assume role stuff. I figured I'd pull it in so if we ever did need to lean on advanced features it'd be there.

The more I think about it, maybe we should remove this to simplify the workflow. What do you think?

Reply:
I think keeping this is fine.